tests/shadow/test_shadow_class.py

```diff
@@ -0,0 +1,463 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for the classical shadows class"""
+# pylint:disable=no-self-use, import-outside-toplevel, redefined-outer-name, unpacking-non-sequence, too-few-public-methods, not-an-iterable, inconsistent-return-statements
+
+import pytest
+
+import pennylane as qml
+import pennylane.numpy as np
+from pennylane.shadows import ClassicalShadow
+from pennylane.shadows import median_of_means, pauli_expval
```

Comment:
Merge imports into one line.
```suggestion
from pennylane.shadows import ClassicalShadow, median_of_means, pauli_expval
```